lib/wallet/src/Cardano/Wallet/Write/Tx.hs

```diff
@@ -896,6 +902,22 @@ toCardanoValue
     -> Cardano.Value
 toCardanoValue = withConstraints (recentEra @era) Cardano.fromMaryValue
 
+--------------------------------------------------------------------------------
+-- PParams
+--------------------------------------------------------------------------------
+
+newtype FeePerByte = FeePerByte Natural
+    deriving (Show, Eq)
+
+feePerByte
+    :: RecentEra era
+    -> Core.PParams (Cardano.ShelleyLedgerEra era)
+    -> FeePerByte
+feePerByte era pp = FeePerByte $ case era of
+    RecentEraConway -> pp ^. #_minfeeA
+    RecentEraBabbage -> pp ^. #_minfeeA
+    RecentEraAlonzo -> pp ^. #_minfeeA
```

Comment:
Suggestion: use a name that more strongly implies that this is a function, where value is returned as a result.

Justification: we can avoid the confusion between `feePerByte` and `feePerByte'` (which look almost identical).

Suggestions:
- `getFeePerByte`
- `getFeePerByteParameter`

```suggestion
getFeePerByte
    :: RecentEra era
    -> Core.PParams (Cardano.ShelleyLedgerEra era)
    -> FeePerByte
getFeePerByte era pp = FeePerByte $ case era of
    RecentEraConway -> pp ^. #_minfeeA
    RecentEraBabbage -> pp ^. #_minfeeA
    RecentEraAlonzo -> pp ^. #_minfeeA
```